index.markdown

```diff
@@ -32,7 +32,10 @@ You can select from these firmware:
 -----
 
 ## ZBDongle-E
-Sonoff Zigbee 3.0 USB Dongle Plus V2 - No hardware flow control  
+Sonoff Zigbee 3.0 USB Dongle Plus V2 - No hardware flow control
+
+***NOTE:*** The latest CP210X Windows driver has known compatibility issues. For devices with the CP2102N serial chip, please install the older version 6.7.0.0 driver: [CP210x VCP Windows](https://www.silabs.com/developers/usb-to-uart-bridge-vcp-drivers?tab=downloads).  
```

Comment:
maybe put a date in here as well. i.e. affects dongles newer than mid-2024.
I think started seeing these issues 3-4 moths ago..

Reply:
I purchased the Dongle with CP2102 in March, and I believe SONOFF had already changed the serial port chip before that.

Reply:
hmm ok, just put 2024 then. there were certainly no wide-spread issues prior to the start of this year!

Reply:
ok

Reply:
This is the batch number that they switched to cp2102 (as printed on back of device)

> New batch of ZBDongle-E (30623XXXXX and above)